nixos/modules/services/monitoring/prometheus/exporters.nix

```diff
@@ -301,6 +302,13 @@ in
         Please specify either 'services.prometheus.exporters.sql.configuration' or
           'services.prometheus.exporters.sql.configFile'
       '';
+    } {
+      assertion = cfg.scaphandre.enable -> (
+        (builtins.elem "intel_rapl_common" config.boot.kernelModules)
+      );
+      message = ''
+        Please enable 'intel_rapl_common' in boot.kernelModules
+      '';
```

Comment:
I think I should also assert that machine runs on Intel or AMD according to the [compatibility](https://hubblo-org.github.io/scaphandre-documentation/compatibility.html) page but I don't know how to do that.

Reply:
I tried to restrict as much as possible the possibility to use Scaphandre without having the proper configuration.